third_party/pugixml-1.5/pugixml.cpp

```diff
@@ -145,6 +151,10 @@ PUGI__NS_BEGIN
 PUGI__NS_END
 #endif
 
+#if (__cplusplus < 201103L)
```

Comment:
The parentheses are not needed here (nor in other instances of this pattern).

Reply:
I removed the parentheses for the `__cplusplus` condtion from Port.hpp, HookGen.cpp, and pugixml.cpp.